addon/index.ts

```diff
@@ -8,7 +8,7 @@ export {
 } from './waiter-manager';
 
 export { default as Token } from './token';
-export { default as TestWaiter } from './test-waiter';
+export { default as AsyncTestWaiter } from './async-test-waiter';
```

Comment:
I'd like to **NOT** export this class _at all_...

Reply:
I'm with you there. This would be a breaking change though, no?